docs/conf.py

```diff
@@ -468,32 +445,8 @@ def get_version(*file_paths):
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {
-    'python': ('https://docs.python.org/3.6', None),
-    'django': ('https://docs.djangoproject.com/en/1.11/', 'https://docs.djangoproject.com/en/1.11/_objects/'),
-    'model_utils': ('https://django-model-utils.readthedocs.io/en/latest/', None),
-}
-
-
-def on_init(app):  # pylint: disable=unused-argument
-    """
-    Run sphinx-apidoc after Sphinx initialization.
-
-    Read the Docs won't run tox or custom shell commands, so we need this to
-    avoid checking in the generated reStructuredText files.
-    """
-    docs_path = os.path.abspath(os.path.dirname(__file__))
-    root_path = os.path.abspath(os.path.join(docs_path, '..'))
-    apidoc_path = 'sphinx-apidoc'
-    if hasattr(sys, 'real_prefix'):  # Check to see if we are in a virtualenv
-        # If we are, assemble the path manually
-        bin_path = os.path.abspath(os.path.join(sys.prefix, 'bin'))
-        apidoc_path = os.path.join(bin_path, apidoc_path)
-    check_call([apidoc_path, '-o', docs_path, os.path.join(root_path, 'edx_developer_docs'),
-                os.path.join(root_path, 'edx_developer_docs/migrations')])
-
-
-def setup(app):
-    """Sphinx extension: run sphinx-apidoc."""
-    event = 'builder-inited' if six.PY3 else b'builder-inited'
-    app.connect(event, on_init)
+# intersphinx_mapping = {
+#     'python': ('https://docs.python.org/3.6', None),
+#     'django': ('https://docs.djangoproject.com/en/1.11/', 'https://docs.djangoproject.com/en/1.11/_objects/'),
+#     'model_utils': ('https://django-model-utils.readthedocs.io/en/latest/', None),
+# }
```

Comment:
Commented out code?

Reply:
Removed this and the rest of the unused config.